### DISCUSSION TITLE
400 MW Solar Power Project at Bhadla, Rajasthan, India (Gold Standard project 1455)

### DISCUSSION THREAD

**Discussion by sichen1234** [opentaps/open-climate-investing#32] [Offsets]
In this thread, I'll look at the 400 MW Solar Power Project at Bhadla, Rajasthan, India on [Gold Standard Marketplace](https://marketplace.goldstandard.org/collections/projects/products/eki-energy-services-gold-standard-foundation-400-mw-solar-power-project-at-bhadla-rajasthan-india), which is in their registry as [Project 1455](https://registry.goldstandard.org/projects/details/1455)

**Comment by sichen1234** [comment]
The main project description document (PDD) is the GS4GG_PDD_SBEnergy_V07-cleanmode.pdf from [SustainCert website](https://platform.sustain-cert.com/public-project/1529)  Key points are:

- This is a straightforward 400 MW utility scale solar project which sells electricity directly back to the main grid.
- The climate impact is calculated as the metered solar electricity generated * emissions factor for the Indian electricity grid.
- Grid emissions factor comes from the [Central Electricity Authority (CEA) of India](https://cea.nic.in/cdm-co2-baseline-database/?lang=en)   
- The PDD calculates an emission factor factor of 0.9638 tCO2e/MWH from Version 14 of CEA emissions factors.  The latest version 17  from CEA has slightly lower emissions factors of 0.91
- The test for additionality is from [CDM](https://cdm.unfccc.int/methodologies/PAmethodologies/tools/am-tool-01-v5.2.pdf/history_view)  The one cited in the PDD is the latest version, 7.0.0
- The PDD makes the following arguments for additionality based on the tool:
  - The IRR for the equity investor is only 3.19% vs an inflation-adjusted benchmark of 13.41% (pages 27 - 29)
  - There are no other solar projects between 200 MW to 600 MW in Rajasthan (pages 31 - 33)
- Monitoring is of the net metered electricity, plus other issues as raised by constituents in the area

**Comment by sichen1234** [comment]
The problem with this project and other large scale renewable energy projects is that they may not be truly additional.  Here are some evidence pointing to that:

- Rajasthan is India's leading state for solar and renewable energy ([Hindustan Times)(https://www.hindustantimes.com/india-news/with-generation-capacity-of-17-040-mw-rajasthan-leads-in-production-of-renewable-energy-101650307992942.html))
- [S&P](https://www.spglobal.com/commodityinsights/en/market-insights/latest-news/energy-transition/022122-reckoning-with-renewables-as-carbon-certifiers-tighten-rules-renewable-energy-may-re-evaluate-options) points out that carbon credit standards are turning away from renewable energy as it has become economically viable on its own.
- Gold Standard held a [consultation in 2018](https://www.goldstandard.org/sites/default/files/documents/re_eligibility_-_for_second_consultation_-_june_2019_.pdf) to re-consider the criteria for renewable energy projects.  It recommended restricting the issuance of carbon credits for renewable energy credits only in the lowest income countries.  
- This in turn became part of the [current Gold Standard Renewable Energy Activity Requirements](https://globalgoals.goldstandard.org/202-ar-renewable-energy-activity-requirements/) under section 2.1.3

So this particular project filed its PDD in August 2017 and began crediting in 2019, right when the standards were changing.

**Comment by sichen1234** [comment]
In addition, there are some important issues with the methodology in general:

- The amount of emissions credited for a renewable energy project is calculated as the amount of energy generated multiplied by the emissions factor.  This is calculated based on annual net metered electricity.  The correct way, however, is to use the actual timing of the renewable energy generation multiplied by the grid emissions factor for those particular times.  Otherwise, we'd over-estimate the  emissions reduction from switching to solar, since there most of solar generation is during peak sun hours already.  This phenomenon is known as the [duck curve](https://www.energy.gov/eere/articles/confronting-duck-curve-how-address-over-generation-solar-energy)  The renewable energy sector is already addressing this issue, with time-stamped energy attribute certificates, EnergyTag standard, Google's 24/7 Clean Energy, and Salesforce's renewable energy procurement policy.  Carbon credits should do the same
- The tests for additionality are not convincing.  The project claims that it is additional because its equity IRR is too low compared to a benchmark when leveraged with debt.  In that case, the equity IRR can be improved by using less debt.  The correct way to look at this is to compare unleveraged IRR of this project versus other comparable projects, especially other solar projects that have gotten off the ground.  Finally, if projects are moving forward with carbon credits being sold at a lower price, then there is a problem with the IRR analysis and its assumptions.
- That brings us to the real test of additionality: Are there other solar projects in the region?  This project claims that it is additional because there are no other solar projects of its size, but this is one of the biggest solar projects in India.  The PDD cited a [Rajasthan government report from 2015](https://energy.rajasthan.gov.in/content/dam/raj/energy/rrecl/pdf/Activities/Solar/4.37%20Details%20of%20commissioned%20Solar%20Projects.pdf) which showed that there were quite a few 25 MW to 40 MW solar projects already.  So unless there is A) some dis-economy from going larger and B) some special benefit from larger projects, there's no reason to believe that this project would not be economical just because it's larger than the others.

**Comment by sichen1234** [comment]
Finally an issue not addressed with the methodology or the PDD for this project is whether the grid utility which buys the power output is claiming it as part its renewable energy.  If so, then the utility, the state of Rajasthan, and the country of India would be reducing its carbon emissions based on this project's output.  That's a good thing, of course, but is the carbon credit then double counted?

**Comment by hai-yr** [comment]
Great analysis.

**Reply by sichen1234** [reply]
Thanks!

**Comment by sichen1234** [comment]
Interestingly enough, an IPO on the Indian stock market was filed for this very solar project: https://mercomindia.com/acmes-ipo-bhadla-solar-project/

I wonder what the financials presented to the investors look like, and whether the investors thought the project was economically viable or not.

**Comment by knagware9** [comment]
Great analysis

**Comment by sichen1234** [comment]
Thanks!

On Thu, Aug 4, 2022 at 6:40 AM Kamlesh Nagware ***@***.***>
wrote:

> Great analysis
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/opentaps/open-climate-investing/discussions/32#discussioncomment-3325483>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANAS4KPMEDCZD6DVPYZVKLVXPB27ANCNFSM5427XDQQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <opentaps/open-climate-investing/repo-discussions/32/comments/3325483@
> github.com>
>
-- 
-----
Si Chen
Open Source Strategies, Inc.

*Why open source and blockchain for carbon accounting?  Video
<https://www.youtube.com/watch?v=eNM7V8vQCg4> and Blog Post
<https://www.opensourcestrategies.com/2022/06/01/why-open-source-carbon-accounting/>*
